config/initializers/bullet.rb

```diff
@@ -14,6 +14,9 @@
       end
 
       Bullet.raise = true if Rails.env.test?
+
+      # Need to eager load :users which is causing it to complain about the join table
+      Bullet.add_whitelist type: :unused_eager_loading, class_name: "Organization", association: :organizations_users
```

Comment:
I previously fixed a warning that we needed eager loading of :users. That did get rid of that warning, but now says we're unnecessarily loading `:organizations_users` (which is the join table). This adds it to a whitelist so we won't be incessantly pinged by an alert that cannot be fixed. 😄